sysadmin/Vagrantfile

```diff
@@ -0,0 +1,187 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+require 'yaml'
+settings = YAML.load_file 'vagrant_settings.yml'
+
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  # required_plugins = %w( vagrant-vbguest vagrant-disksize )
+  # _retry = false
+  # required_plugins.each do |plugin|
+  #       unless Vagrant.has_plugin? plugin
+  #           system "vagrant plugin install #{plugin}"
+  #          _retry=true
+  #       end
+  # end
+
+  # if (_retry)
+  #       exec "vagrant " + ARGV.join(' ')
+  # end
+
+  config.vm.define "opencontext" do |node|
```

Comment:
So far, I've put in a lot of work on getting `opencontext_predb` working (the configuration that relies on a pre-existing database and solr instance). But I'd like to make sure `opencontext` configuration, which builds up a database and solr database from scratch also works.

Reply:
That sounds great!!